pkgs/applications/misc/slic3r/prusa3d.nix

```diff
@@ -72,6 +76,8 @@ stdenv.mkDerivation rec {
     XMLSAX
   ]);
 
+  NLOPT = "${nlopt}";
```

Comment:
What does this do? Please document.

Reply:
I have added a comment, as well as two others on the fixes during the patchPhase.

Reply:
On second thought, is `gtest` in nativeBuildInputs correct? As far as I understand it, nativeBuildInputs is also used for crosscompiling, and in that case should ` gtest`  be for the host or the target platform?

Reply:
Tests are somewhat of a special case: They can only be run when host == target anyways. Since they are run at "runtime", they technically should go to the `buildInputs`. Sometimes I also define `checkInputs` and then do `buildInputs = [ ... ] ++ checkInputs` for clarity.

Reply:
I've adapted this approach. Thanks for the hint!